### PR TITLE
Refactor/comments

### PR DESCRIPTION
**Pull request by datka-mcoder**

### CODE REVIEW

File: src/main/java/kg/peaksoft/taskTrackerb6/api/CommentApi.java [hunk 1]
```diff
@@ -21,17 +22,16 @@ public class CommentApi {
     private final CommentService commentService;
 
     @Operation(summary = "Save comment", description = "Save new comment")
-    @PostMapping("card/{id}")
-    public CommentResponse saveComment(@PathVariable Long id,
+    @PostMapping("card/{cardId}")
```
Comment (i-Coderr) [inline src/main/java/kg/peaksoft/taskTrackerb6/api/CommentApi.java:25]: Remove card

File: src/main/java/kg/peaksoft/taskTrackerb6/api/CommentApi.java [hunk 2]
```diff
@@ -41,8 +41,8 @@ public SimpleResponse deleteComment(@PathVariable Long id){
     }
 
     @Operation(summary = "Get all comments", description = "Get all comments by card id")
-    @GetMapping("card/{id}")
-    public List<CommentResponse> findAllCommentsByCardId(@PathVariable Long id){
-        return commentService.findAllComments(id);
+    @GetMapping("card/{cardId}")
```
Comment (i-Coderr) [inline src/main/java/kg/peaksoft/taskTrackerb6/api/CommentApi.java:44]: Remove card

File: src/main/java/kg/peaksoft/taskTrackerb6/db/model/Comment.java [hunk 3]
```diff
@@ -25,16 +24,26 @@ public class Comment {
     @Column(length = 10000)
     private String text;
 
-    private LocalDateTime createdAt;
+    private String createdAt;
```
Comment (i-Coderr) [inline src/main/java/kg/peaksoft/taskTrackerb6/db/model/Comment.java:27]: Why String?

File: src/main/java/kg/peaksoft/taskTrackerb6/dto/request/CommentRequest.java [hunk 4]
```diff
@@ -10,4 +10,5 @@
 public class CommentRequest {
 
     private String text;
+    private String addCommentDate;
```
Comment (i-Coderr) [inline src/main/java/kg/peaksoft/taskTrackerb6/dto/request/CommentRequest.java:13]: Why date is String?

File: src/main/java/kg/peaksoft/taskTrackerb6/dto/response/CommentResponse.java [hunk 5]
```diff
@@ -1,23 +1,32 @@
...
 
 @Getter
 @Setter
+@AllArgsConstructor
 public class CommentResponse {
 
     private Long id;
     private String text;
-    private LocalDateTime createdAt;
+    private String createdAt;
```
Comment (i-Coderr) [inline src/main/java/kg/peaksoft/taskTrackerb6/dto/response/CommentResponse.java:15]: ?

**Review by i-Coderr — COMMENTED** [review]
Why are so many refactored classes in  PR?

File: src/main/java/kg/peaksoft/taskTrackerb6/db/model/Comment.java (hunk 3 above)
Comment (datka-mcoder) [reply src/main/java/kg/peaksoft/taskTrackerb6/db/model/Comment.java:27]: Фронтенд бизге String кылып бергиле, createdAt маанисин биз озубуз беребиз дейт

**Review by i-Coderr — APPROVED** [review]
Ok

File: src/main/java/kg/peaksoft/taskTrackerb6/db/model/Comment.java (hunk 3 above)
Comment (i-Coderr) [reply src/main/java/kg/peaksoft/taskTrackerb6/db/model/Comment.java:27]: Ok

**Pull request merged**